spikeinterface/extractors/neoextractors/mearec.py

```diff
@@ -48,15 +46,13 @@ def __init__(self, file_path, use_natural_unit_ids=True):
         self._kwargs = {'file_path': str(file_path), 'use_natural_unit_ids': use_natural_unit_ids}
 
 
-def read_mearec(file_path, locs_2d=True, use_natural_unit_ids=True):
+def read_mearec(file_path, use_natural_unit_ids=True):
```

Comment:
Why did you remove this locs_2d here ? It will make fail many many NB and test in many place no ?